### ISSUE TITLE
Context menu

### ISSUE THREAD

**Issue by emanb29** [Silk-GUI/Silk#16]
A good quick thing to fix would be making a custom context menu. Since ideally this will be for nodeOS, but will also likely be implemented in webkit-based applications, it's worth looking for a cross-platform solution. I know [html5 has an API for this (contextmenu attr/menu elem), though at the moment only firefox supports it.](http://www.w3.org/html/wg/drafts/html/master/semantics.html#attr-contextmenu)

**Comment by zodern** [comment]
Silk will run in chrome on NodeOS, so the html5 won't work for now.

There are many jquery plugins.  https://github.com/medialize/jQuery-contextMenu looks like a good option.

**Comment by emanb29** [comment]
Yeah, I assumed we'd be running on some variant of Blink/webkit. jQuery-contextMenu could be an option, though the style is a bit inconsistent with the rest of Silk
